onedal/svm/svm.py

```diff
@@ -45,7 +42,7 @@ class SVMtype(Enum):
     nu_svr = 3
 
 
-class BaseSVM(BaseEstimator, metaclass=ABCMeta):
```

Comment:
Please close https://github.com/intel/scikit-learn-intelex/pull/1727 in case of green ci checks in this PR